css/styles.css

```diff
@@ -1,23 +1,51 @@
 @import url('https://fonts.googleapis.com/css?family=Poppins');
 
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   font-family: 'poppins', sans-serif;
-  width: auto;
-  height: auto;
 }
 
 .navbar {
+  width: 100%;
   display: flex;
   text-decoration: none;
   flex-wrap: wrap;
-  justify-content: space-around;
   align-items: center;
   color: #6070ff;
   font-size: 18px;
+  transition: 250ms height ease-in-out;
+  padding: 24px;
+  position: fixed;
+  justify-content: space-between;
+  background: #fff;
 }
 
 .box-menu-1 {
-  display: none;
+  width: 100%;
+  background-color: #6070ffe3;
+  margin: 0;
+  position: fixed;
+  top: 0;
+  left: -100%;
+  height: 100%;
+  box-sizing: border-box;
+  padding: 40px;
+  transition: 250ms height ease-in-out;
+}
```

Comment:
- [ ] Please let's follow the design guidelines and implement the blur on the mobile menu as shown in the template 🙏 
**HINT:** Use `backdrop-filter: blur(specify the blur in px)` 😎 
## Figma Design
![image](https://user-images.githubusercontent.com/46839548/188737545-ba29e04d-fb13-4fc1-8d80-dbcdcefa6f82.png)

## Your Design
![image](https://user-images.githubusercontent.com/46839548/188737436-41e24196-b4c4-4ca4-bf8d-42b4e28c32a2.png)